sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentOptions.cs

```diff
@@ -5,13 +5,14 @@ namespace Azure.AI.TextAnalytics
 {
     /// <summary>
     /// Options that allow callers to specify details about how the operation
-    /// is run and what information is returned from it by the service.
+    /// is run. For example set model version, whether to include statistics, and execute opinion mining.
```

Comment:
I would include opinion mining first in this list of examples to have it jump out more